src/index.ts

```diff
@@ -0,0 +1,33 @@
+import {ApolloServer} from 'apollo-server'
+import {types as typeDefs, resolvers} from './graphql'
+import {prisma, Option} from './utils'
+import  IsAuthenticatedDirective  from './Directives'
+
+
+const options: Option =  {
+  port: Number(process.env.GRAPHQL_PORT),
+  endpoint: String(process.env.GRAPHQL_PLAYGROUND),
+  playground: String(process.env.GRAPHQL_PLAYGROUND),
+  debug: Boolean(process.env.DEBUG)
+}
+
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  schemaDirectives: {
+    isAuthenticated: IsAuthenticatedDirective
+  },
+  context: (request): object => {
+    return {
+      request,
+      prisma
+    }
+  },
+});
+
+server.listen( options.port ).then(({ url }) => {
```

Comment:
Parenthesises should have no spaces around, only curly braces have spaces around.

```suggestion
server.listen(options.port).then(({ url }) => {
```